app/src/main/res/values/colors.xml

```diff
@@ -17,4 +17,6 @@
     <color name="colorDarkGray">#404040</color>
     <color name="colorLightGray">#d0d0d0</color>
     <color name="category_card_background">#fdfdfd</color>
+
+    <color name="submissionPromptTextColor">#131313</color>
```

Comment:
Ahh I was trying to keep it consistent with the other camelCase ones we let in there before. I prefer snake case for xml res too